src/usr/local/www/firewall_rules_edit.php

```diff
@@ -1217,7 +1217,7 @@ function build_if_list() {
 	'Protocol',
 	$pconfig['proto'],
 	array(
-		'any' => gettext('any'),
+		'any' => gettext('Any'),
```

Comment:
The word "any" is not a proper noun, abbreviation or acronym.  Don't think it should typically be capitalized in this context of a protocols list.